cloud/services/compute/network.go

```diff
@@ -103,6 +104,26 @@ func (s *Service) DeleteNetwork() error {
 		}
 	}
 
+	// Delete routes associated with network
+	filterString := fmt.Sprintf("description=k8s-node-route name=%s-*", s.scope.Name())
+	routeList, err := s.routes.List(s.scope.Project()).Filter(filterString).Do()
+	if err != nil {
+		return errors.Wrapf(err, "failed to list routes for the cluster")
+	}
+
+	for _, route := range routeList.Items {
+		if strings.HasSuffix(route.Network, network.Name) {
+			s.scope.Info("deleting route ", "route:", route.Name)
+			op, err := s.routes.Delete(s.scope.Project(), route.Name).Do()
+			if err != nil {
+				return errors.Wrapf(err, "failed to delete routes")
+			}
+			if err := wait.ForComputeOperation(s.scope.Compute, s.scope.Project(), op); err != nil {
+				return errors.Wrapf(err, "failed to delete routes")
```

Comment:
this err msg is same as above. maybe -> failed to wait delete routes?

Reply:
++ack